src/main/java/com/GG/T9AgeCombat/models/Unit.java

```diff
@@ -44,13 +44,22 @@
     Integer selection;
     Integer standardBearer;
     boolean hasMusician;
+    Integer musician;
+    @NonFinal
+    Integer reRollToHitLessThan;
+    @NonFinal
+    Boolean hasReRollToWound;
+    @NonFinal
+    Boolean hasReRollArmorSave;
+    @NonFinal
+    Boolean hasReRollLeadership;
+    @NonFinal
+    Integer toHitBonus;
+    @NonFinal
+    Integer extraRanks;
     List<SpecialRule> specialRuleList;
     List<Equipment> equipmentList;
 
-    public void setPendingWounds(int pendingWounds) {
-        this.pendingWounds += pendingWounds;
-    }
-
```

Comment:
Did you meant to remove this?

Reply:
I did not... and I thought i added it back in?